src/presentation/pages/Verticals/HallOfFame/useHallOfFame.ts

```diff
@@ -2,9 +2,11 @@ import { useCallback, useEffect, useState } from 'react';
 import { useGetNftsPrimaryMutation } from 'infrastructure/services/nft/NftService';
 import { NFT } from 'app/interfaces/NFT/NFT';
 
-export const useVertical = () => {
-  const [getNftsPrimary, { isError, isLoading, isSuccess }] = useGetNftsPrimaryMutation();
+export const useHallOfFame = () => {
+  const [getNftsPrimary, { isError, isLoading, isSuccess, data }] = useGetNftsPrimaryMutation();
   const [nfts, setNfts] = useState<NFT[]>([]);
+  const [tier, setTier] = useState<string>('');
```

Comment:
Can we use a type here? in order to restrict values when setTier is called. Same applies for setLegend as we have a fixed  set of values for them.

Reply:
Done!